frontend/public/config.yml

```
@@ -34,26 +34,10 @@ collections:
-            }
-          - {
-              label: 'Hero_Button2_Text',
-              name: 'hero_button2_text',
-              widget: 'string',
-            }
-          - {
-              label: 'Hero_Button2_Link',
-              name: 'hero_button2_link',
-              widget: 'string',
-            }
+          - { label: 'Hero_Button1_Text', name: 'hero_button1_text', widget: 'string' }
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)